### ISSUE TITLE
CPU-only version

### ISSUE THREAD

**Issue by leeyang** [lupoglaz/TorchProteinLibrary#32]
Very nice work! But is it possible to install TorchProteinLibrary on a CPU only (without cuda) environment?

**Comment by lupoglaz** [comment]
Yes, it is possible, but some layers are hopelessly slow on cpu. I'll add an option to compile cpu-only version.

**Comment by leeyang** [comment]
Hi! Any progress? I find your method is much faster than another methods even though I implemented it simply using raw Pytorch autograd.

**Comment by lupoglaz** [comment]
I am working on it, but it's a huge pita: I have cuda kernels and cpu side in the same extension modules. Might take a week/two more.

**Comment by gkementzidis** [comment]
Hi, are there any updates on the CPU-only version? Thanks.
